src/sidebar/store/modules/annotations.js

```diff
@@ -545,6 +543,35 @@ function savedAnnotations(state) {
   });
 }
 
+/**
+ * @typedef AnnotationsStore
+ *
+ * // Actions
+ * @prop {typeof addAnnotations} addAnnotations
+ * @prop {typeof clearAnnotations} clearAnnotations
+ * @prop {typeof focusAnnotations} focusAnnotations
+ * @prop {typeof hideAnnotation} hideAnnotation
+ * @prop {typeof removeAnnotations} removeAnnotations
+ * @prop {typeof unhideAnnotation} unhideAnnotation
+ * @prop {typeof updateAnchorStatus} updateAnchorStatus
+ * @prop {typeof updateFlagStatus} updateFlagStatus
+ 
```

Comment:
There is a blank line here and some trailing space. Maybe install something like [this extension](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces#review-details) to highlight trailing whitespace in your IDE?

Reply:
installed, thanks!